controllers/controllers.js

```diff
@@ -0,0 +1,35 @@
+const {getAllCustomers, getOneCustomer, addNewCustomers} = require('./models/models')
+
+const getAllCustomersController = async (request, response) => {
+   try{const allCustomers = await getAllCustomers()
+
+    return response.send(allCustomers)
+   } catch(error){
+        return response.sendStatus(500)
```

Comment:
It will be nice for the errors to also return an error message. So when you get a 500 status you know what caused the issue.